### ISSUE TITLE
Accessibility Testing|| On-boarding pages images need to add texts to support talk back

### ISSUE THREAD

**Issue by poojaustad** [UstadMobile/UstadMobile#341]
**Expected behavior**
On ustadmobile android app - the onboarding pages contain images, On talk back, these images should explain the image content

**Describe the bug**
On ustadmobile android app -**talk back** not responding to the onboarding page images, 

**To Reproduce**
1. Go to the 'Ustadmobile- get started 'page
2. Click on 'the images'
3. Talkback not respond to the image on the on-boarding page
![2022-06-23_14h27_00](https://user-images.githubusercontent.com/78338755/175278961-d2fd05e4-eb37-47e1-8600-973e508db14a.png)
![2022-06-23_14h28_44](https://user-images.githubusercontent.com/78338755/175278970-d95b5e11-ea70-431f-91dd-9522123c296e.png)

**Comment by mikedawson** [comment]
The illustrations shown are purely decorative, so there shouldn't be any talkback for those. The logos (e.g. supporter logos) should have talkback on them).
